### PR TITLE
Add users

### PR DESCRIPTION
**Pull request by oatalabi**

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
+    templateUrl: 'app/partials/loginUser.html',
+    controller: 'userCtrl'
+  })
+  .otherwise({
+    redirectTo: '/'
   });
-});
\ No newline at end of file
+
+}]);
```
Comment (bosunolanrewaju) [inline app/app.js:25]: Missing newline here. You can configure your text editor if SublimeText or RubyMine to automatically add the newline.

File: app/app.js [hunk 2]
```diff
@@ -1,14 +1,25 @@
 var app = angular.module('bucketlistApp',['ngRoute']);
 
-app.config(function($routeProvider){
-  $routeProvider.when('/', {
+app.config(['$routeProvider', '$httpProvider', function($routeProvider, $httpProvider){
+  $routeProvider
```
Comment (bosunolanrewaju) [inline app/app.js:4]: `routeProvider` ? why not UI-Router? https://github.com/angular-ui/ui-router

File: app/services/userService.js [hunk 3]
```diff
@@ -1,13 +1,27 @@
...
+        headers:{
+          'Content-Type': 'application/x-www-form-urlencoded'
+        },
+        data: loginDetails,
+        url: 'http://kickmylist.herokuapp.com/api/v1/auth/login'
+      });
     },
     logout: function(){
-      return $http.post('http://kickmylist.herokuapp.com/api/v1/users');
+      return $http.get('http://kickmylist.herokuapp.com/api/v1/users');
```
Comment (bosunolanrewaju) [inline app/services/userService.js:24]: Is the API expecting a get request for logout?

**Comment by bosunolanrewaju** [comment]
Just a couple of things about formatting, newlines and indentation.

File: app/services/userService.js (hunk 3 above)
Comment (oatalabi) [inline app/services/userService.js:24]: Yes it is, that's what was given in the checkpoint for Bucketlist Api

File: app/app.js (hunk 2 above)
Comment (oatalabi) [inline app/app.js:4]: I prefer using ngRoute, I've only used UI-Router once. I will take a look at it again

File: app/app.js (hunk 1 above)
Comment (oatalabi) [inline app/app.js:25]: OK!

**Pull request closed without merging**